webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+
+module.exports = {
```

Comment:
@douglas-johnson Thank you for getting these files combined into the same build!